src/types/CharData.d.ts

```diff
@@ -0,0 +1,62 @@
+import firebase from "firebase";
+import DocumentReference = firebase.firestore.DocumentReference;
+
+export default interface CharData {
+  id?: string; // キャラクターの一意のID
+  user?: string; // キャラクター作成者
+  createDate?: number; // 作成日 経過ミリ秒の文字列でいいのかな？
+  modifiedDate?: string; // 更新日 キャラ一覧の並び替えなどで使う用
+  userRef?: DocumentReference; //?
+  profile?: {
+    // キャラクターの基本情報
+    name?: string; // 名前
+    isDead?: boolean; // 生存状況
+    avatarUrl?: string; // プロフィール画像のURL
+    occupation?: string; // 職業
+    backstory?: string; // 説明文 改行を許容
+    height?: number; // 身長
+    age?: number;
+    weight?: number;
+    sex?: string; // 性別 [male|female]
```

Comment:
TRPGだしmale|femaleでは収まらなさそうなので、コメント削ったほうが良さそう。
```suggestion
    sex?: string; // 性別
```

Reply:
修正しました